.github/workflows/integration-tests.yaml

```
@@ -0,0 +1,67 @@
+        env:
+          BUNDLE_IMG: ${{ env.IMAGE_TAG_BASE }}-bundle:${VERSION_TAG}
+        run: |
+          make bundle-build
+          make bundle-push
+
+      - name: Create catalog image
+        env:
+          CATALOG_IMG: ${{ env.IMAGE_TAG_BASE }}-catalog:${VERSION_TAG}
+          BUNDLE_IMG: ${{ env.IMAGE_TAG_BASE }}-bundle:${VERSION_TAG}
+        run: |
+          make catalog-build 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)